var/spack/repos/builtin/packages/helics/package.py

```diff
@@ -142,21 +142,27 @@ def cmake_args(self):
 
         # HELICS shared library options
         args.append(
-            "-DHELICS_DISABLE_C_SHARED_LIB={0}".format("OFF" if "+c_shared" in spec else "ON")
+            "-DHELICS_DISABLE_C_SHARED_LIB={0}".format(
+                "OFF" if spec.satisfies("+c_shared") else "ON"
+            )
```

Comment:
These and below appear to be breaking with conventions earlier in this file, and for consistentcy might be better as:
```suggestion
        args.append(from_variant("HELICS_DISABLE_C_SHARED_LIB", "c_shared"))
```

Reply:
I guess it's opposite, with this change, it would disable c shared libs when the variant is set.

Reply:
Right, good catch. We don't have an inversion in `define_from_variant`, so let's just leave it as is then.